cu_shim.py

```
@@ -0,0 +1,77 @@
+    "python3 -m pytest -q",
+    "flake8 termnet/termnet/ tests/ scripts/",
+    "git status --porcelain",
+}
+
+
+def run(cmd, cwd=None, timeout=30):
+    """Execute a shell command safely."""
+    try:
+        # For simple commands in SAFE set, use shell=False with split
+        if cmd in SAFE or cmd.startswith("echo ") or cmd.startswith("cat "):
+            p = subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)